src/Containers/Microsoft.NET.Build.Containers/AuthHandshakeMessageHandler.cs

```diff
@@ -263,8 +263,10 @@ private async Task<(AuthenticationHeaderValue, DateTimeOffset)?> TryTokenGetAsyn
             var tokenResponse = await base.SendAsync(message, cancellationToken).ConfigureAwait(false);
             if (!tokenResponse.IsSuccessStatusCode)
             {
-                throw new UnableToAccessRepositoryException(_registryName);
+                await tokenResponse.LogHttpResponseAsync(_logger, cancellationToken).ConfigureAwait(false);
             }
+            tokenResponse.EnsureSuccessStatusCode();
+            _logger.LogTrace("Received '{statuscode}'.", tokenResponse.StatusCode);
```

Comment:
This change should be reverted - the more specific exception is used by other parts of the tooling to give users specific instructions/signal specific failure states.